pkg/commands/pull_request.go

```diff
@@ -35,18 +35,42 @@ func NewService(typeName string, repositoryDomain string, siteDomain string) *Se
 	switch typeName {
 	case "github":
 		service = &Service{
-			Name:           repositoryDomain,
-			PullRequestURL: fmt.Sprintf("https://%s%s", siteDomain, "/%s/%s/compare/%s?expand=1"),
+			Name: repositoryDomain,
+			PullRequestURL: func(owner string, repository string, from string, to string) string {
+				if to == "" {
+					urlFormat := fmt.Sprintf("https://%s%s", siteDomain, "/%s/%s/compare/%s?expand=1")
```

Comment:
Why double `fmt.SprintF`?

Reply:
Fixed

Reply:
there hasn't changed anything yet here i think you ment to say this on another comment